recipes/default.rb

```diff
@@ -35,27 +35,34 @@
   notifies :reconfigure, 'chef_server_ingredient[supermarket]'
 end
 
-if node['supermarket_package']['package_source']
-  pkgname = ::File.basename(node['supermarket_package']['package_source'])
+if node['supermarket_omnibus']['package_url']
```

Comment:
hey @binamov is there a reason you renamed this attribute?  This change would break all existing users of this attribute.